integrations/hubspot/nango.yaml

```diff
@@ -218,12 +218,20 @@ integrations:
                     - oauth
             update-contact:
                 description: Updates a single contact in Hubspot
-                output: CreateUpdateContactOutput
+                output: SuccessResponse
```

Comment:
Why did the update contact output change?

Reply:
Apologies, that was accidental, Fixing it Asap